zadaci/sa-kolokvijuma/2020/RA/T34/G6/zadatak.rst

```diff
@@ -0,0 +1,43 @@
+Iz zadate ulazne datoteke automobili.txt učitati statički niz struktura, 
+pri čemu se struktura ``automobil_st`` sastoji od sledećih polja:
+
+- marka (jedna reč, do 20 karaktera)
+- model (jedna reč, do 20 karaktera)
+- vrsta servisa (jedna reč, do 20 karaktera)
+- kubikaža (prirodan broj)
+- budžet vlasnika (prirodan broj)
+
+Vrsta servisa koji je potreban za automobil može biti MALI ili VELIKI. 
+Budžet vlasnika predstavlja količinu novca koju je je vlasnik automobila
+spreman da izdvoji za željeni servis.
+
+Stvarna cena servisiranja automobile zavisi od vrste servisa i 
+od kubikaže automobila. Cena servisiranja data je u tabeli ispod:
+
++---------------------+--------------------+----------------------+
+| Kubikaža            | Cena malog servisa | Cena velikog servisa |
++=====================+====================+======================+
+| kubikaza<1300       | 5 000              | 10 000               |
++---------------------+--------------------+----------------------+
+| 1300<=kubikaza<1800 | 8 000              | 15 000               |
++---------------------+--------------------+----------------------+
+| kubikaza>=1800      | 12 000             | 25 000               |
++---------------------+--------------------+----------------------+
+
+Na standardni izlaz (terminal) ispisati sve automobile za koje se cena 
+servisiranja uklapa u budžet vlasnika.
+
+Primer poziva:
+
+  .. code:: bash
```

Comment:
Please don't indent RST directives, as this places them inside quotation blocks and messes up the books HTML/PDF output.

Reply:
Updated.